optional_plugins/varianter_yaml_to_mux/avocado_varianter_yaml_to_mux/__init__.py

```diff
@@ -484,12 +492,17 @@ def initialize(self, config):
 
         # Extend default multiplex tree of --mux-inject values
         for inject in config.get("yaml_to_mux.inject"):
-            entry = inject.split(':', 3)
+            entry = inject.split(':', 2)
```

Comment:
I'm not sure why this changed.

Reply:
This is because, we need only key and value to be split. If it is split for more parts, then the `:` in the mux value is also split. So, urls (for example) are alo split and not read completely.